javascript.ru/2.12.1/2.12.1.js

```diff
@@ -0,0 +1,3 @@
+﻿if ("0") {
+	alert('Привет');
+}
```

Comment:
Add another ways to test:
1. if (0)
2. if ("")
3. if (undefined)

Reply:
Done 62c6e67.

Reply:
Done c251d36.